parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -67,7 +67,7 @@ installDeps() {
 
 updateAptWithMicrosoftPkg() {
     if [[ $(isARM64) == 1 ]]; then
-        if [ "${UBUNTU_RELEASE}" == "22.04" ]; then
+        if [[ "${UBUNTU_RELEASE}" == "22.04" || "${UBUNTU_RELEASE}" == "24.04" ]]; then
```

Comment:
@anujmaheshwari1 I remember I had to make this change when I was first working on this. Do you know if that's not necessary anymore?

Reply:
it likely still is!